packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1322,3 +1322,6 @@ export const MEETING_PERMISSION_TOKEN_REFRESH_REASON = 'ttl-join';
 
 // constant for named media group type
 export const NAMED_MEDIA_GROUP_TYPE_AUDIO = 1;
+
+// constant for statsAnalyzer report.callingServiceType
+export const CALLING_SERVICE_TYPE = 'LOCUS' as const;
```

Comment:
Should we change this to an enum and list out all the available options in [CallingServiceType.raml](https://sqbu-github.cisco.com/WebExSquared/event-dictionary/blob/05fb283bf3d11cd3a684512bbec3967e5489ac29/types/CallingServiceType.raml)?

I know we will only be setting this as `LOCUS` for multistream meetings but I think there are a few benefits to having all the options listed out:

1. It feels more complete.
2. It provides more context in terms of what these values mean (i.e. seeing the other options makes it easier to understand what `LOCUS` means here).
3. It makes it easier to change use different values in the future. For example, Webex Calling will be using this soon, and they would probably want to use the `WEBEXCALLING` option.